third-party/CMakeLists.txt

```diff
@@ -173,6 +176,10 @@ endif()
 include(GetBinaryDeps)
 
 if(WIN32)
+  # curl.exe, ca-bundle.crt, tidy.exe, swiss.exe
+  GetBinaryDep(TARGET wintools
+    INSTALL_COMMAND ${CMAKE_COMMAND} -E copy_directory . ${DEPS_INSTALL_DIR}/bin)
+
```

Comment:
@equalsraf Thanks for the help, it's extremely appreciated. That is the direction I wanted to go, I ran into some build error and this was a temporary hack. Extracted to https://github.com/neovim/neovim/pull/6105